vignettes/admiral.Rmd

```diff
@@ -292,21 +250,23 @@ created by calling `use_ad_template()`, e.g.,
 
 ```r
 use_ad_template(
-  adam_name = "adsl",
-  save_path = "./ad_adsl.R"
+adam_name = "adsl",
+save_path = "./ad_adsl.R"
 )
 ```
 
 A list of all available templates can be obtained by `list_all_templates()`:
 
 ```{r}
+library(admiral)
 list_all_templates()
 ```
 
 # Support
 
 Support is provided via the [admiral Slack
-channel](https://app.slack.com/client/T028PB489D3/C02M8KN8269).
+channel](https://app.slack.com/client/T028PB489D3/C02M8KN8269). Additionally,
+please feel free to raise issues in our [GitHub repository](https://github.com/pharmaverse/admiral/issues).
 
 # See also
 
```

Comment:
Should we add a link to the "Programming Concepts and Conventions" vignette?

Reply:
good suggestion, added